.github/workflows/test.yaml

```
@@ -542,7 +588,7 @@ jobs:
         run: |
           cd packages/solara-enterprise
           openssl sha256 dist/*
-          hatch publish
+          # hatch publish
```

Comment should also be removed here

```suggestion
          hatch publish
```